tests/AndreeaHasel.spec.js

```diff
@@ -1,9 +1,7 @@
-// @ts-check
 const { test, expect } = require("@playwright/test");
```

Comment:
import { test, expect } from "@playwright/test";
Try use this syntax, it's more readable